python/tvm/driver/tvmc/frontends.py

```diff
@@ -130,7 +130,9 @@ def load(self, path, shape_dict=None):
         input_shapes = {name: x.shape for (name, x) in zip(model.input_names, inputs)}
         if shape_dict is not None:
             input_shapes.update(shape_dict)
-        return relay.frontend.from_keras(model, input_shapes, layout="NHWC")
+        layout = kwargs.get("layout", "NHWC")
```

Comment:
To make this more clear I would do `kwargs.setdefault("layout", "NHWC")` instead -- this does the same thing (I think!)